explorer/client/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -195,8 +202,13 @@ function GroupView({ results }: { results: resultType }) {
                                     </span>
                                 </div>
                             </div>
-                        );
-                    }
+                        </div>
+                    );
+                })}
+                {uniqueTypes.length % 3 === 2 && (
```

Comment:
so this will only show when there are 5, 8, 11, 14 etc. types ?  
what's the reasoning behind this logic ?

Reply:
thinking about this more - i guess this is some sort of filler object ? could we give this test a descriptive variable name?

Reply:
Done